shared_objects/lib1.c

```diff
@@ -1,3 +1,11 @@
+/**
+ * A basic example showing how CHERI 
```

Comment:
Can we format these to a slightly wider width -- 76, 80, or 100 chars are all reasonable choices IMHO, but does waste a lot of horizontal screen space in these days of widescreen monitors :/

Reply:
done